mlir/test/Dialect/Vector/vector-emulate-masked-load-store.mlir

```diff
@@ -0,0 +1,105 @@
+// RUN: mlir-opt %s --test-vector-emulate-masked-load-store | FileCheck %s
+
+module attributes {
+  spirv.target_env = #spirv.target_env<
+    #spirv.vce<v1.0, [Shader], [SPV_KHR_storage_buffer_storage_class]>, #spirv.resource_limits<>>
```

Comment:
Do we need the SPIR-V dialect to test this? It would be better to make vector dialect tests not to depend on the spirv dialect.